tool/pull-latest-mspec-spec

```diff
@@ -3,16 +3,14 @@
 # Assumes all commits have been synchronized to https://github.com/ruby/spec
 # See spec/mspec/tool/sync/sync-rubyspec.rb
 
-rm -rf spec/mspec
-git clone --depth 1 https://github.com/ruby/mspec.git spec/mspec
-commit=$(git -C spec/mspec log -n 1 --format='%h')
-rm -rf spec/mspec/.git
-git add spec/mspec
-git commit -m "Update to ruby/mspec@${commit}"
+update() {
+    rm -rf "spec/$1"
+    git clone --depth 1 "https://github.com/ruby/${2-$1}.git" "spec/$1"
```

Comment:
What's `${2-$1}` ? $2 if there is a $2 otherwise $1?
I think it would be simpler to read by always passing 2 arguments to `update()`.